src/components/widgets/FileWidget.js

```diff
@@ -60,10 +60,6 @@ function extractFileInfo(dataURLs) {
 }
 
 class FileWidget extends Component {
-  defaultProps = {
```

Comment:
On the bottom it shows : 

```

 FileWidget.defaultProps = {
   autofocus: false,
 };
 ```

so it seems the defaultprops are not the same, this is strange ...

Reply:
`multiple: false` should be added there, since `multiple` is not a required property

Reply:
The reason I initially left it out was to do the simplest possible change. When defaultProps is set as an instance property, they are ignored, meaning that if we add `multiple: false` to the defaultProps, it is small change in the functionality. If the maintainers think we should fix both problems on a single PR, I'm happy to make the change 🙂

Reply:
Ok, I think we should rather fix the other problem later, to not break backwards-compatibility.